src/fragments/lib-v1/storage/js/download.mdx

```diff
@@ -0,0 +1,118 @@
+There are three ways of getting data that was previously uploaded:
+
+## Generate a download URL
+
+Get a presigned URL of a stored file and expiration of URL. You can specify [access `level`](https://docs.amplify.aws/lib-v1/storage/configureaccess/q/platform/js) of file
+
+```typescript
+import { getUrl } from 'aws-amplify/storage';
+
+const getUrlResult: {
+	url: URL;
+	expiresAt: Date;
+} = await getUrl({
+        key: filename,
+        options: { accessLevel: 'private' | 'protected' | 'guest' , // defaults to `guest` 
+        targetIdentityId?: string, // id of another user, if `level: protected`
+        validateObjectExistence: false,  // defaults to false
+        expiresIn: 20 // validity of the URL, in seconds. defaults to 900 (15 minutes) and maxes at 3600 (1 hour)
```

Comment:
Looks good for now, but I'd like to add a advanced use case section later showing how to get a long-live(7 days) presigned URL